pages/annotation.vue

```diff
@@ -1,39 +1,63 @@
-          <b-card-text>
-            <component
-              :is="page.component"
-              :columns="columnToCategoryMap"
-              :dataTable="dataTable.original"
-              @remove:column="writeColumn($event)"
-              @update:dataTable="writeTable($event)"
-            ></component>
-          </b-card-text>
-        </b-tab>
-      </b-tabs>
-    </no-ssr>
-  </b-container>
+
+    <b-container fluid>
+
+        
+            <!--
+                TODO: revisit the client-side render solution or remove this comment
+                The v-for statement below was causing a mismatch between client-side and server-side
+                DOM. In particular, the first element in "pages" (Age) was rendered twice. The error message was:
+                Vue warn]: The client-side rendered virtual DOM tree is not matching server-rendered content.
+                This is likely caused by incorrect HTML markup,
+                for example nesting block-level elements inside <p>, or missing <tbody>.
+                Bailing hydration and performing full client-side render.
+
+                The best answer I found online was this pretty useless stackoverflow answer:
+                https://stackoverflow.com/a/61375490/1302009 suggesting that this might be due to some
+                async getting of Array data.
+                I forced this block to be rendered client-side only for now and that fixed it for now
+                See: https://nuxtjs.org/docs/features/nuxt-components/#the-client-only-component
+            -->
+            <no-ssr>
+            <!-- This gives us built-in keyboard navigation! -->
+            <b-tabs pills card vertical>
+                <!--      TODO: hardcode the pages and just toggle visibility based on state-->
+                <b-tab v-for="page in pages" :title="page.title" :key="page.id">
+                <b-card-text>
+                    <component
+                    :is="page.component"
+                    :columns="columnToCategoryMap"
+                    :dataTable="dataTable.original"
+                    :pageData="pageData"
+                    @remove:column="writeColumn($event)"
+                    @update:dataTable="writeTable($event)"
+                    ></component>
+                </b-card-text>
+                </b-tab>
+            </b-tabs>
+            </no-ssr>
+        
+
+        <b-row>
+
+            <b-col cols="9"></b-col>
+
+            <!-- Button to proceed to the next page -->
+            <!-- Only enabled when at least one annotation table write has been done -->
+            <b-col cols="3">
+                <b-button
+                    class="float-right"
+                    :disabled="!pageData.download.accessible"
+                    :to="'/' + pageData.download.location"
+                    :variant="nextPageButtonColor">
+                    Next step: Categorize columns
```

Comment:
```suggestion
                    Next step: Review and download the harmonized data
```

Reply:
> suggestion: let's call the button "Download" (I suggested inline)

Oops! Left over from my copy-paste from the landing page. Will replace.

> comment: let's remve the pageData props (see comment)

Good catch! Use of `pageData` here is leftover from a previous implementation where it was necessary as a prop for components since the 'Confirm and Upload' button(s) reside inside them – and was being used to proceed to the download page. Passing it through the annotation components is no longer necessary.